boot/zephyr/main.c

```diff
@@ -76,6 +76,53 @@ static void do_boot(struct boot_rsp *rsp)
     __set_MSP(vt->msp);
     ((void (*)(void))vt->reset)();
 }
+
+#elif defined(CONFIG_XTENSA)
+
+static void copy_img_to_SRAM(int slot, unsigned int hdr_offset)
+{
+    const struct flash_area *fap;
+    int area_id;
+    int rc;
+    unsigned char *dst = (unsigned char *)0xBE030000 + hdr_offset;
```

Comment:
doses this magic value not possible to being get from somewhere in DT or Kconfig?

Reply:
I have explored this option of fetching it from DT but it doesn't seem to be straight forward. It's possible to get it from Kconfig but would really like to do it from DT. Can that be deferred for a while? I can add a macro for now.